files.go

```diff
@@ -0,0 +1,108 @@
+package pack
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+	"github.com/pkg/errors"
+
+	"github.com/buildpack/pack/builder"
+	"github.com/buildpack/pack/internal/paths"
+)
+
+// ReadBuilderConfigFile reads a builder configuration from the file path provided
+func ReadBuilderConfigFile(path string) (builder.Config, error) {
```

Comment:
Minor: Feels like now that these are external there should be some tests associated with them for certain guarantees. Given that this is mostly simply reorganization we could probably live without them.

Reply:
That makes sense. Since it's still tested indirectly via other tests, I'm OK deferring.

Reply:
I agree with @jromero  that we should add unit tests if we are exposing this as part of the library API.

Reply:
@ekcasey A private helper is not a problem. It would just have to be duplicated. pack can't import the commands package, and commands cant use it if it lives privately in pack. I could create a third package to decouple, but feels a little tedious.

Reply:
@ekcasey OK, I'll write some tests to get us started. They probably won't be as exhaustive as the cases we go through for create builder, but we can slowly migrate them over perhaps.

Reply:
Move to internal package and added tests